markdown/developers/sync.md

```diff
@@ -355,3 +355,61 @@ When new releases of `nf-core/tools` and it's associated template are released,
 be created to merge updates in to your pipeline for you.
 
 That's it, you're done! **Congratulations!**
+
+# Fixing a broken TEMPLATE branch
+
+Sometimes mistakes can occur during resolving of conflicts during a template merge, typically via the GitHub interface, where the changes during the conflict
+resolution is committed to `TEMPLATE` rather than the `dev` branch itself.
```

Comment:
```suggestion
If merge conflicts are resolved via the GitHub interface instead of after pulling changes to a fork as described above, the commit history from the `dev` branch will be merged into `TEMPLATE`.
```